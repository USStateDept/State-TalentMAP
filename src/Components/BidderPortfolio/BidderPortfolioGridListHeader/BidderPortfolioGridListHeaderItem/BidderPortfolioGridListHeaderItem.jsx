import PropTypes from 'prop-types';
import FontAwesome from 'react-fontawesome';

const BidderPortfolioGridListHeaderItem = ({ className, content }) => (
  <div className={`usa-grid-full portfolio-grid-list-header-item ${className}`}>
    {content} <FontAwesome name="sort" />
  </div>
);

BidderPortfolioGridListHeaderItem.propTypes = {
  className: PropTypes.string,
  content: PropTypes.node.isRequired,
};

BidderPortfolioGridListHeaderItem.defaultProps = {
  className: '',
};

export default BidderPortfolioGridListHeaderItem;
