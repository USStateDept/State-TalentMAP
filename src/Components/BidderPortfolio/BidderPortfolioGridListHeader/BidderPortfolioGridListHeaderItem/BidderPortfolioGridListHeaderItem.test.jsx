import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import BidderPortfolioGridListHeaderItem from './BidderPortfolioGridListHeaderItem';

describe('BidderPortfolioGridListHeaderItemComponent', () => {
  it('is defined', () => {
    const wrapper = shallow(<BidderPortfolioGridListHeaderItem content="test" />);
    expect(wrapper).toBeDefined();
  });

  it('matches snapshot', () => {
    const wrapper = shallow(<BidderPortfolioGridListHeaderItem content="test" />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
