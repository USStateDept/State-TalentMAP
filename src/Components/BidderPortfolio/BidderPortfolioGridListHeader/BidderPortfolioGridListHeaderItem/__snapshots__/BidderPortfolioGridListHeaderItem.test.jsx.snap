// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BidderPortfolioGridListHeaderItemComponent matches snapshot 1`] = `
<div
  className="usa-grid-full portfolio-grid-list-header-item "
>
  test
   
  <FontAwesome
    name="sort"
  />
</div>
`;
